examples/asr/transcribe_speech.py

```diff
@@ -151,10 +152,18 @@ def main(cfg: TranscriptionConfig) -> TranscriptionConfig:
     else:
         # get filenames from manifest
         filepaths = []
+        if os.stat(cfg.dataset_manifest).st_size == 0:
+            logging.error(f"The input dataset_manifest {cfg.dataset_manifest} is empty. Exiting!")
+            return None
+
         with open(cfg.dataset_manifest, 'r') as f:
+            partial_audio = True
             for line in f:
                 item = json.loads(line)
+                if "offset" not in item or "duration" not in item:
```

Comment:
This should be an if x in manifesf and y in manifest: situation, not an or. Most manifests will have duration but not offset.

Reply:
Not really. if anyone is missing, then partial_audio=False and we will use `audio_filepath` and `transcribe` method

Reply:
We generally have the 3 values in the manifest - audio_filepath. text and duration. Here the or for "offset" will fail and set partial_audio = False which is fine.
The logic should be - partial audio is false by default, but if both args are provided, then set to true - its the inverse of this.